<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>35</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(69, 83, 100);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <widget class="QToolButton" name="btnMenu">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>35</width>
            <height>27</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 0px;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../images/menu_24px.png</normaloff>../images/menu_24px.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>24</height>
           </size>
          </property>
         </widget>
         <widget class="QToolButton" name="btnPlay">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>50</y>
            <width>35</width>
            <height>27</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 0px;
</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../images/play_24px.png</normaloff>../images/play_24px.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>24</height>
           </size>
          </property>
         </widget>
         <widget class="QToolButton" name="btnReset">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>100</y>
            <width>35</width>
            <height>27</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius: 0px;
</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../images/restart_24px.png</normaloff>../images/restart_24px.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="menuLabelContainer" native="true">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(84, 104, 122);
color: rgb(255, 255, 255);</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;&amp;nbsp;&amp;nbsp;Menu&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lblPlay">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>    Demarrer la simulation</string>
       </property>
      </widget>
      <widget class="QLabel" name="lblRestart">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>    Redemarrer la simulation</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="pandaContainer" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>280</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Environnement</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QListWidget" name="envList"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Propriétés</string>
          </attribute>
          <widget class="QWidget" name="formLayoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>271</width>
             <height>33</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="nomObject">
              <property name="text">
               <string>Nom de l'objet</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QWidget" name="nomObjectWidget" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>855</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouveau"/>
    <addaction name="actionEnregistrer"/>
    <addaction name="actionEnregistrer_2"/>
    <addaction name="actionEnregistrer_sous"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionMettre_en_pause"/>
    <addaction name="actionRedemarrer_la_simulation"/>
   </widget>
   <widget class="QMenu" name="menucamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionRapprocher"/>
    <addaction name="actionEloigner"/>
   </widget>
   <widget class="QMenu" name="menuEnvironnement">
    <property name="title">
     <string>Environnement</string>
    </property>
    <addaction name="actionNouveau_drone"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuSimulation"/>
   <addaction name="menucamera"/>
   <addaction name="menuEnvironnement"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNouveau">
   <property name="text">
    <string>Nouveau</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="text">
    <string>Ouvrir...</string>
   </property>
  </action>
  <action name="actionEnregistrer_2">
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </action>
  <action name="actionEnregistrer_sous">
   <property name="text">
    <string>Enregistrer sous ...</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionDemarrer_la_simulation">
   <property name="text">
    <string>Demarrer la simulation</string>
   </property>
  </action>
  <action name="actionMettre_en_pause">
   <property name="text">
    <string>Demarrer la simulation</string>
   </property>
  </action>
  <action name="actionRedemarrer_la_simulation">
   <property name="text">
    <string>Redemarrer la simulation</string>
   </property>
  </action>
  <action name="actionRapprocher">
   <property name="text">
    <string>Rapprocher</string>
   </property>
  </action>
  <action name="actionEloigner">
   <property name="text">
    <string>Eloigner</string>
   </property>
  </action>
  <action name="actionNouveau_drone">
   <property name="text">
    <string>Ajouter drone</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
